tools/downloader/README.md

```diff
@@ -231,32 +235,33 @@ configuration file and exit:
 
 ```
 $ ./TOOL.py --print_all
-Sphereface
 action-recognition-0001-decoder
 action-recognition-0001-encoder
 age-gender-recognition-retail-0013
-alexnet
-brain-tumor-segmentation-0001
-ctpn
-deeplabv3
-densenet-121
-densenet-121-tf
+driver-action-recognition-adas-0002-decoder
```

Comment:
is it an actual list that is shown by --print_all? does it mean it prints all the Intel models in alphabetical order and then public ones? don't we need to print the full path then (imagine what could happen in case if due to some reason there is a model under the same name both under intel and public)?

Reply:
> is it an actual list that is shown by --print_all? does it mean it prints all the Intel models in alphabetical order and then public ones?

Yes and yes.

> don't we need to print the full path then (imagine what could happen in case if due to some reason there is a model under the same name both under intel and public)?

The config loading code will abort if there are two models with the same name.